scipy/stats/_distn_infrastructure.py

```diff
@@ -2589,6 +2589,16 @@ def fit(self, data, *args, **kwds):
         >>> loc1, scale1 = norm.fit(x)
         >>> loc1, scale1
         (0.92087172783841631, 2.0015750750324668)
+
+        Using a custom optimizer:
+
+        >>> def optimizer_NM(func, x0=[0.6, 1.6], args=(), disp=0):
```

Comment:
where did you take these argument values from / why did you choose these values?

Reply:
Thanks for reviewing. I replaced the normal example with the beta, as normal does not use the ``optimizer`` argument https://github.com/m-maggi/scipy/blob/f964b3c812eb911a18f30eb26c146108e020862e/scipy/stats/_continuous_distns.py#L393-L399

Furthermore I added 
> The optimizer must return the estimated parameters.

In the doctoring.